test/unit/test_describe.py

```diff
@@ -33,29 +33,35 @@ def test_describe(
         if expid not in _EXPIDS:
             continue
         exp = autosubmit_exp(expid)
-
+    
```

Comment:
Unnecessary blank spaces (some IDEs like VS Code & PyCharm will delete these blank spaces when the file is saved -- if any line is modified, causing Git changes)